lms/djangoapps/courseware/testutils.py

```diff
@@ -71,8 +71,9 @@ def setup_course(self, default_store=None):
         with self.store.default_store(default_store):
             self.course = CourseFactory.create(**self.course_options())  # pylint: disable=attribute-defined-outside-init
             chapter = ItemFactory.create(parent=self.course, category='chapter')
+            section = ItemFactory.create(parent=chapter, category='sequential')
             self.html_block = ItemFactory.create(  # pylint: disable=attribute-defined-outside-init
-                parent=chapter,
+                parent=section,
```

Comment:
What is the benefit of this change?  Does the test still pass if the section isn't part of a chapter?  That is to say, does this change hide a potential bug for course authors?  (These are not rhetorical questions.)